### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by alex-phillips**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]
	

 

------------------------------

We welcome all PR’s though this doesn’t guarantee it will be accepted.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	$BUILD_PACKAGES && \
+ rm -rf \
+	/root/.cache \
+	/tmp/*
+
+# copy local files
+COPY root/ /
+
+# ports and volumes
+EXPOSE 8000
```
Comment (nemchik) [inline Dockerfile:71]:
```suggestion
EXPOSE 8000
VOLUME /config
```

File: Dockerfile.aarch64 [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	$BUILD_PACKAGES && \
+ rm -rf \
+	/root/.cache \
+	/tmp/*
+
+# copy local files
+COPY root/ /
+
+# ports and volumes
+EXPOSE 8000
```
Comment (nemchik) [inline Dockerfile.aarch64:71]:
```suggestion
EXPOSE 8000
VOLUME /config
```

File: Dockerfile.armhf [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	$BUILD_PACKAGES && \
+ rm -rf \
+	/root/.cache \
+	/tmp/*
+
+# copy local files
+COPY root/ /
+
+# ports and volumes
+EXPOSE 8000
```
Comment (nemchik) [inline Dockerfile.armhf:71]:
```suggestion
EXPOSE 8000
VOLUME /config
```

File: Dockerfile (hunk 1 above)
Comment (alex-phillips) [reply Dockerfile:71]: I believe we quit explicitly defining volumes in Dockerfiles as these don't do anything for the container other than create additional images if nothing is defined that build up and need to be cleaned up manually over time.

**Pull request merged**